### PR TITLE
PWGMM: Added process function for centrality in MC gen

### PR DESCRIPTION
**Pull request by gbencedi**

### CODE REVIEW

File: PWGMM/Mult/Tasks/dndeta-mft.cxx [hunk 1]
```diff
@@ -74,49 +74,79 @@ struct PseudorapidityDensityMFT {
...
+
+  Configurable<float> cfgPhiCut{"cfgPhiCut", 0.1f,
+                                "Cut on azimuthal angle of MFT tracks"};
+
   //-----need access to CCDB to get the reweighting histogram
   Service<ccdb::BasicCCDBManager> ccdb;
-  Configurable<std::string> path{"ccdb-path", "Users/s/sherrman/My/Object", "base path to the ccdb object"};
-  Configurable<std::string> url{"ccdb-url", "http://alice-ccdb.cern.ch", "url of the ccdb repository"};
-  Configurable<int64_t> nolaterthan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
+  Configurable<std::string> path{"ccdb-path", "Users/s/sherrman/My/Object",
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/dndeta-mft.cxx:93]: These CCDB-related configurables are not used in the task, as I understand this is a leftover for the vertex Z re-weighting that was done at some point. You can remove them.

Comment (gbencedi) [reply PWGMM/Mult/Tasks/dndeta-mft.cxx]: Thanks for noticing, Anton. I removed them.

File: PWGMM/Mult/Tasks/dndeta-mft.cxx [hunk 2]
```diff
@@ -1,6 +1,6 @@
-// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
-// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
-// All rights not expressly granted are reserved.
+// Copyright 2020-2022 CERN and copyright holders of ALICE O2.
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/dndeta-mft.cxx:1]: Please also make sure the the copyright headers are not reformatted from their original state.

**Review by aalkin — CHANGES_REQUESTED** [review]

**Review by aalkin — APPROVED** [review]

**Comment by aalkin** [comment]
Please do not apply Megalinter automatic fixes, they seem to not working as intended at the moment.

**Pull request merged**